ios/app/src/Launch Screen.storyboard

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this file change necessary? can we reuse existing LaunchScreen?

Reply:
don't want to mess with the original launchscreen.